crates/sncast/src/starknet_commands/helpers/fee.rs

```diff
@@ -1,12 +1,14 @@
 use anyhow::{bail, ensure, Result};
 use clap::{Args, ValueEnum};
+use sncast::handle_account_factory_error;
+use starknet::accounts::{AccountDeploymentV1, AccountDeploymentV3, AccountFactory};
 use starknet::core::types::FieldElement;
 
 #[derive(Args, Debug)]
 pub struct FeeArgs {
     /// Token that transaction fee will be paid in
     #[clap(long)]
-    pub fee_token: FeeToken,
+    pub fee_token: Option<FeeToken>,
```

Comment:
Why did this become an `Option`?

Reply:
Because you can infer the token from the transaction version, the user doesn't need to specify the token.

Reply:
Oh I see it now, thanks